pkg/server/node.go

```diff
@@ -1155,6 +1155,9 @@ func (n *Node) Batch(
 	tenantID, ok := roachpb.TenantFromContext(ctx)
 	if !ok {
 		tenantID = roachpb.SystemTenantID
+	} else {
```

Comment:
Why `else`? Don't we want the tenant tag to always be present, albeit =1 for the system tenant?